src/markdown/inline/types.ts

```diff
@@ -29,6 +29,7 @@ export interface IInlineCode extends IToken {
   type: 'inlineCode';
   value: string;
   wrap: string;
+  language?: string;
```

Comment:
Rename this to `lang`.

```suggestion
  lang?: string;
```